src/main/main.c

```diff
@@ -311,8 +311,14 @@ void init(void)
     spiInit(SPIDEV_1);
 #endif
 #ifdef USE_SPI_DEVICE_2
+#ifdef OMNIBUSF4
```

Comment:
Sorry @nathantsoi I led you up the garden path - though I think there is still merit in hardware revisions - and I have some further changes to introduce to make them easier shortly.